packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/SetBasicAccessRequirementFields.test.tsx

```diff
@@ -91,31 +88,39 @@ function setUp(props: SetBasicAccessRequirementFieldsProps) {
 describe('SetBasicAccessRequirementFields', () => {
   beforeEach(() => jest.clearAllMocks())
   beforeAll(() => server.listen())
-  afterEach(() => server.restoreHandlers())
+  afterEach(() => server.resetHandlers())
   afterAll(() => server.close())
 
-  test('does not show text instructions for self-signed AR', () => {
-    const { buttons } = setUp({
-      accessRequirement: mockSelfSignAccessRequirement,
-      onSaveComplete,
+  test('does not show text instructions for self-signed AR', async () => {
+    const { buttons } = await setUp({
+      accessRequirementId: mockSelfSignAccessRequirement.id.toString(),
+      onSave,
+      onError,
     })
     expect(buttons.deleteTextInstructions).toBeNull()
+    await waitForMarkdownSynapseToGetWiki()
```

Comment:
Need to wait MarkdownSynapse calls to finish, otherwise there will be pending requests that impact subsequent tests